test/functional/activation_key/subscriptions_test.rb

```diff
@@ -54,11 +54,14 @@ module HammerCLIKatello
       run_cmd(%w(activation-key add-subscription --id 1 --subscription-id 3))
     end
 
-    it 'allows a subscription to be added by name to an activation key' do
+    it 'allows a subscription to be added by name to an activation key with org label' do
```

Comment:
Do you know what code change actually made this necessary?

Reply:
@jturel it looks like it was this guy

https://github.com/Katello/katello/pull/9151

Reply:
I can add a comment to the pr if you want?

Reply:
I was curious because I didn't see any changes in activation_keys_controller but your link has made it perfectly clear. Ty!